pytket/extensions/quantinuum/backends/api_wrappers.py

```diff
@@ -341,9 +344,14 @@ def retrieve_job_status(
         job_url = f"{self.url}job/{job_id}"
         # Using the login wrapper we will automatically try to refresh token
         id_token = self.login()
+        params = {}
         if use_websocket or (use_websocket is None and self.use_websocket):
-            job_url += "?websocket=true"
-        res = self.session.get(job_url, headers={"Authorization": id_token})
+            params.update({"websocket": "true"})
+        if request_raw_results:
+            params.update({"results_format": "raw"})
```

Comment:
Why `params.update()` rather than `params["websocket"] = "true"` and `params["results_format"] = "raw"`?

They do the same thing, but `.update` looks a little odd when it's just adding one field.